source/AI.cpp

```diff
@@ -369,7 +369,12 @@ void AI::Step(const PlayerInfo &player)
 		shared_ptr<Ship> parent = it->GetParent();
 		if(parent && parent->IsDestroyed())
 		{
-			parent.reset();
+			// NPCs that lose their fleet leader follow the next most senior ship
+			// (which is the player, unless the NPC is "uninterested").
```

Comment:
This comment is incorrect, or at least confusing - it should say something like, "NPCs that lose their fleet leader should follow the *leader's parent*."